data_handler.py

```diff
@@ -140,12 +144,42 @@ def main(self, datalog, cpm, cpm_err, this_start, this_end, counts):
         self.manager.data_log(datalog, cpm, cpm_err)
 
         if self.manager.test:
-            self.test_send(cpm, cpm_err)
+            self.send_to_memory(cpm, cpm_err)

```

Comment:
Hmm, why this change? If it's in test mode, I think you can still use `test_send()` which just prints a notification.